src/main/java/com/minecolonies/coremod/blocks/BlockHutField.java

```diff
@@ -132,20 +133,21 @@ public boolean isPassable(final IBlockAccess worldIn, final BlockPos pos)
     }
 
     //todo: remove once we no longer need to support this
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)